### ISSUE TITLE
Add a page for "Julia resources at your university/company/research center"

### ISSUE THREAD

**Issue by Keno** [JuliaLang/www.julialang.org#872]
I often find that myself telling people about other groups at their own institutions that are already using Julia, or compute clusters available to them that have Julia installed, etc. I think it would be good to have a list of such resources and contact people (only if they're willing of course) to connect users with the appropriate resources at their own institutions.

**Comment by Keno** [comment]
Or maybe also "existing Julia users in your field".

**Comment by Keno** [comment]
Initial Content:
- MIT: Contact Julia Lab
- Google: There's an internal mailing list
- Microsoft: We know there's a number of internal users, not sure if anybody wants to be listed publicly, don't think there's a mailing list (maybe just have people reach out to me or somebody if they want to be connected)
- NERSC: Julia is supported on Cori, support is available for NERSC users

just off the top of my head to get a few things started.

**Comment by Keno** [comment]
NVIDIA has a dedicated Julia person also, though NVIDIA people seem to be able to find him fine, I assume they have some sort of internal directory. Might be good to list also though.

**Comment by Keno** [comment]
Same story with Intel.

**Comment by ChrisRackauckas** [comment]
What about allowing people to declare themselves campus champions? Alan is the campus champion for MIT, etc.? I know some HPC resources do that so people know who to contact.

**Comment by Keno** [comment]
Yes, that's the idea - though I want to make it slightly broader to also allow things like mailing lists or even just links to things like internal jupyterhub cluster that people may have access to even if they don't know it (e.g. I think a lot of the European and Canadian universities do).

**Comment by zdroid** [comment]
@ChrisRackauckas What makes one a campus champion?

@Keno It may be good to contact the people behind https://fluxml.ai/, as they already have a section "Researchers, users and developers of Flux" and may be able to help in finding out the resources you're looking for.

**Comment by ChrisRackauckas** [comment]
>@ChrisRackauckas What makes one a campus champion?

Someone who's willing to be a campus champion 😄 . It's just a name to put up there so that everyone who's interested in Julia at a particular institution has a contact, and whoever is willing to put their name up can be one. It doesn't have to be unique, and surely if there are two names they will know each other by essence of someone had to put their name second, and thus it's already a nice way to get a growing collaboration.

**Comment by zdroid** [comment]
> Someone who's willing to be a campus champion.

Really interesting idea.

I think putting this idea into effect would really make the collaboration grow well, especially on universities without much Julia users. For example, at the University of Belgrade (where I will enroll soon) and its affiliated academic institutions almost all scientific research is exclusively done in C/C++ and Python. So someone interested in Julia would be stuck with those options, without knowing anyone who could help with working in Julia.

There's also a social effect, that universities / researchers will also want to have their name on that list after they've seen the others do the same. All in all, would be good to see this. :smile:

**Comment by ViralBShah** [comment]
Sounds like a good idea!

**Comment by logankilpatrick** [comment]
The next question is how can we implement this in the site without just making one big list. I’m guessing we would need some JS

**Comment by ViralBShah** [comment]
I thought just make it big list. People can just search on the page using the browser's find.

**Comment by zdroid** [comment]
@logankilpatrick @ViralBShah Something similar to the format of https://government.github.com/community/ would be appropriate. What do you think? I'd also advise its use of tiles with images (similar to what we have for Editors and IDEs on the front page).

**Comment by ViralBShah** [comment]
Getting images for all those orgs and maintaining it to look uniform and nice is a lot of work. Also, we want to make it trivial for people to add themselves to that list.

**Comment by ViralBShah** [comment]
My take is let's do something simple and see if people find it useful. The problem is that unless the list is really long, it looks really bad (like only 10 institutions have Julia champions or something).

**Comment by ChrisRackauckas** [comment]
Post on Discourse and we might get like 50 in a day.

**Comment by zdroid** [comment]
@ViralBShah Why does it have to be orgs? If we are talking about universities, every has its own logo. Seeing the logo will help potential Julia users find their colleagues who are using Julia more easily.

> The problem is that unless the list is really long, it looks really bad (like only 10 institutions have Julia champions or something).

I think that would be remedied by this:
> There's also a social effect, that universities / researchers will also want to have their name on that list after they've seen the others do the same.

**Comment by ViralBShah** [comment]
Sorry for being ambiguous - I was referring to real organizations- Universities, companies, etc. Yeah, if they have a github org, we can use that.

**Comment by zdroid** [comment]
@ViralBShah So, what is your final say on the format of the page, should it be similar to https://government.github.com/community/?

**Comment by ViralBShah** [comment]
My suggestion is do the simple thing for now and let's focus effort on the other parts of the website. Let's build some content here and then revisit the presentation a few weeks later.

**Comment by zdroid** [comment]
Is there some specific part that you have in mind? What is proposed here has great potential for the user base growth.

**Comment by ViralBShah** [comment]
I meant the various PRs and issues that are open. It would be good to see those through. Each of the existing "tabs" in the menu is strange in its own way. So having a common design across the site - etc.

**Comment by ViralBShah** [comment]
I think the way to do this is to perhaps set up a google sheet and advertise heavily on discourse and slack and see how many people sign up. If we can get about a hundred or so orgs going, it may be worth having something on the website. Otherwise, a sparse list is worse than no list.
